ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -294,6 +294,48 @@ def test_numpy_instance_argsort(
     )
 
 
+@handle_cmd_line_args
+@given(
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+    keepdims=st.booleans(),
+    where=np_frontend_helpers.where(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.ndarray.mean"
```

Comment:
The test fails with `AttributeError: module 'ivy.functional.frontends.numpy.ndarray' has no attribute 'mean'`. I think the `fn_name` should be `ivy.functional.frontends.numpy.ndarray.ndarray.mean`. 
You can check out errors by clicking `Details` next to 
`test-frontend-numpy / run-nightly-tests (pull_request)`. Then you should expand `Run Frontend Tests` and search for your test's name.